Sources/AWSLambdaEvents/Utils/DateWrappers.swift

```diff
@@ -80,10 +80,12 @@ public struct RFC5322DateTimeCoding: Decodable {
 
     public init(from decoder: Decoder) throws {
         let container = try decoder.singleValueContainer()
-        let dateString = try container.decode(String.self)
+        let string = try container.decode(String.self)
+        let bracket = string.firstIndex(of: "(") ?? string.endIndex
+        let dateString = String(string[string.startIndex ..< bracket])
```

Comment:
I would love to see a comment here why we are doing this.

Reply:
could we potentially save an allocation by making the string a var and rewriting it only if we find a bracket?

Reply:
both of them are done

Reply:
Like